test/src/main/java/org/apache/accumulo/test/functional/ThriftMaxFrameSizeIT.java

```diff
@@ -56,50 +62,95 @@ class TestDefault extends TestMaxFrameSize {
     TestDefault() {
       serverType = ThriftServerType.getDefault();
     }
+
+    @Test
+    public void testDefaultServerFrameSize() throws Exception {
+      AtomicBoolean succeeded = new AtomicBoolean(false);
+      assertThrows(AssertionFailedError.class,
+          () -> assertTimeoutPreemptively(Duration.ofSeconds(30),
+              () -> testMaxFrameSizeLargerThanDefault(succeeded)));
+      assertFalse(succeeded.get());
+    }
```

Comment:
The succeeded boolean is only ever false when the method that sets it is interrupted... which also would throw the assertion failure. So, adding that boolean doesn't actually add more information... it's always false on the interrupt case, and it's always true on the non-interrupt case. So, it's just a proxy for whether or not we used the assertTimeoutPreemptively method. It can be removed.